test/integration_tests/integrationtest_scanner_api.cpp

```diff
@@ -69,20 +64,77 @@ using namespace ::testing;
 using namespace std::chrono_literals;
 using namespace pilz_testutils;
 
-ACTION_P(OpenBarrier, barrier)
+static MonitoringFrameMsg createValidMonitoringFrameMsg()
 {
-  barrier->release();
+  return MonitoringFrameMsg(TenthOfDegree(0), TenthOfDegree(1), 0, { 1, 2, 3, 4, 5 });
 }
 
-ScannerConfiguration createScannerConfig()
+struct PortHolder
 {
-  return ScannerConfiguration(
-      HOST_IP_ADDRESS, HOST_UDP_PORT_DATA, HOST_UDP_PORT_CONTROL, SCANNER_IP_ADDRESS, SCAN_RANGE, DIAGNOSTICS_ENABLED);
-}
+  PortHolder& operator++()
+  {
+    data_port_host = (data_port_host + 1) % MAX_DATA_PORT_HOST;
+    if (data_port_host == 0)
+    {
+      data_port_host = MIN_DATA_PORT_HOST;
+    }
```

Comment:
Could this be simplified as:
```suggestion
    data_port_host = std::max((data_port_host + 1) % MAX_DATA_PORT_HOST, MIN_DATA_PORT_HOST);
```

Reply:
same for the next 3 I gues

Reply:
Already fixed.